gnosis/eth/django/tests/test_models.py

```diff
@@ -146,3 +147,40 @@ def test_keccak256_field(self):
         ):
             with transaction.atomic():
                 Keccak256Hash.objects.create(value=value_hex_invalid)
+
+    def test_serialize_Keccak256Field_to_json(self):
```

Comment:
Use snake case, even if you are referring a class name, same with the other test methods 😉

Reply:
Done [here](https://github.com/safe-global/safe-eth-py/pull/399/commits/cc9303c3b3161ca0fffa287661d764d3e33b334c)